pkgs/tools/misc/coreutils/default.nix

```diff
@@ -11,7 +11,7 @@ assert selinuxSupport -> libselinux != null && libsepol != null;
 with { inherit (stdenv.lib) optional optionals optionalString optionalAttrs; };
 
 let
-  self = stdenv.mkDerivation rec {
+  self = rec {
```

Comment:
Good catch!
The flipside is, to override `FORCE_UNSAFE_CONFIGURE` before building anything, it has to be a plain attribute set...